### DISCUSSION TITLE
I'm not able to see kibana in my tpot menu few days after deploying the tpots

### DISCUSSION THREAD

**Discussion by JebilJC** [telekom-security/tpotce#1423] [Q&A]
So, I deployed some tpots a week ago, it was workign fine the first few days, but when I check it now, i cant find kibana in the menu. Can someone please help me out, its pretty urgent.
![image](https://github.com/telekom-security/tpotce/assets/47811802/da877c03-0910-4637-9215-f3060875a57d)

**Comment by t3chn0m4g3** [comment]
#1379

**Comment by PLACIDEM** [comment]
I have exactly the same issue  still troubleshooting  and  waiting  the solution
